docs/patterns/errorpages.rst

```diff
@@ -97,3 +97,30 @@ An example template might be this:
       <p>What you were looking for is just not there.
       <p><a href="{{ url_for('index') }}">go somewhere nice</a>
     {% endblock %}
+
+
+Handling API Errors with Abort
+------------------------------
+
+When using Flask for web APIs, handling errors is as simple as the previous examples but with minor changes.
+
+Register the error handler::
+
+    from flask import jsonify
+
+    @app.errorhandler(404)
+    def resource_not_found(e):
+        # if passing in an exception directly, you may need to convert it to a string
+        return jsonify(error=e)
```

Comment:
As far as I know, `json.dumps` won't be able to serialize an `Exception` object, so this won't work.

Reply:
You're right. That was the comment in the example. I'll update the code to be a little more clear.

Reply:
Added more clarity.